package_plugin.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+pyuic5 ui_files/config.ui -o crowd_anki/config/config_ui.py
```

Comment:
as mentioned above the pyuic5 should be part of dev dependencies, and the installation of them locally should probably be done in `fetch_dependencies`

Reply:
Alright dev it is. Think I have the right one now, after looking into it.